src/DynamoWPFCLI/Program.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using Dynamo.Applications;
+using Dynamo.Models;
+using Dynamo.ViewModels;
+using Dynamo.Wpf.ViewModels.Watch3D;
+
+namespace DynamoWPFCLI
+{
+    internal class Program
+    {
+        [STAThread]
+        static internal void Main(string[] args)
+        {
+
+            try
+            {
+                var cmdLineArgs = StartupUtils.CommandLineArguments.Parse(args);
+                var locale = StartupUtils.SetLocale(cmdLineArgs);
+                var model = StartupUtils.MakeModel(true);
+                var viewModel = DynamoViewModel.Start(
+                    new DynamoViewModel.StartConfiguration()
+                    {
+                        DynamoModel = model,
+                        Watch3DViewModel = new DefaultWatch3DViewModel(null, new Watch3DViewModelStartupParams(model))
+                        {
+                            Active = false,
+                            CanBeActivated = false
+                        }
+                    });
+
+                var runner = new CommandLineRunnerWPF(viewModel);
+                runner.Run(cmdLineArgs);
+                
+            }
+            catch (Exception e)
+            {
+                try
+                {
+                    DynamoModel.IsCrashing = true;
+                    Dynamo.Logging.Analytics.TrackException(e, true);
```

Comment:
hmm, do we want to log crashes for the CLI?

Reply:
We already do it in the other CLI. I just follow the same pattern. Do we want to change it?

Reply:
nope, lets keep it then.